src/teleop/gui/src/components/OdometryReading.vue

```diff
@@ -0,0 +1,72 @@
+<template>
+    <!--
```

Comment:
Delete this comment

Reply:
I made this comment deliberately, I think we should make a map store that works for all the GUIs since currently the odometry reading vue file uses only the auton store